### PR TITLE
[fix] update dictionary 

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by harkiratsm** [comment]
Now I really got to know about petrification and depetrification.

File: utils/manifests/utils.go [hunk 1]
```diff
@@ -227,6 +227,7 @@ func useDictionary(input string, invert bool) (string, bool) {
 		"caBundle":             "CA Bundle",
 		"mtls":                 "mTLS",
 		"mTLS":                 "mTLS",
+		"fsGroupPolicy":        "fSGroup Policy",
```
Comment (leecalcote) [inline utils/manifests/utils.go:230]: @Revolyssup @harkiratsm is there a negative ramification caused by changing "additionalProperties" to "Additional Properties"?

Comment (harkiratsm) [reply utils/manifests/utils.go:230]: Yeah it may cause problem related to rjsf schema itself.

**Comment by leecalcote** [comment]
Hmm. Let's close this for now and circle back at a future date.

**Pull request closed without merging**